airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -471,6 +574,164 @@ definitions:
+              property_limit_type: characters
+              property_limit: 15000
+              record_merge_strategy:
+                type: GroupByKeyMergeStrategy
+                key: id
+      paginator:
+        $ref: "#/definitions/cursor_paginator"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - results
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["updatedAt"]
+            value: "{{ record.get('updatedAt') or record['createdAt'] }}"
+      - type: CustomTransformation
+        class_name: source_hubspot.components.HubspotFlattenAssociationsTransformation
+      - type: CustomTransformation
+        class_name: source_hubspot.components.NewtoLegacyFieldTransformation
+        field_mapping:
+          hs_date_entered_: "hs_v2_date_entered_"
+          hs_date_exited_: "hs_v2_date_exited_"
+          hs_time_in_: "hs_v2_latest_time_in_"
+      - type: DpathFlattenFields
+        field_path:
+          - properties
+        key_transformation:
+          type: KeyTransformation
+          prefix: properties_
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+      is_client_side_incremental: true
+    schema_loader:
+      $ref: "#/definitions/deals_schema_loader"
+
+  deals_incremental_stream:
+    primary_key:
+      - id
+    $ref: "#/definitions/stream_base"
+    name: deals
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/base_requester"
+        path: /crm/v3/objects/deal/search
+        http_method: POST
+        fetch_properties_from_endpoint:
+          type: PropertiesFromEndpoint
+          property_field_path:
+            - name
+          retriever:
+            type: SimpleRetriever
+            requester:
+              $ref: "#/definitions/base_requester"
+              path: /properties/v2/deal/properties
+            decoder:
+              type: JsonDecoder
+            record_selector:
+              type: RecordSelector
+              extractor:
+                type: DpathExtractor
+                field_path: []
+        request_body_json:
+          limit: 100
+          sorts:
+            - propertyName: hs_object_id
+              direction: ASCENDING
+          filters:
+            - propertyName: hs_lastmodifieddate
+              operator: GTE
+              value: "{{ format_datetime(stream_interval.start_time, '%ms') }}"
+            - propertyName: hs_lastmodifieddate
+              operator: LTE
+              value: "{{ format_datetime(stream_interval.end_time, '%ms') }}"
+            - propertyName: hs_object_id
+              operator: GTE
+              value: "{{ next_page_token['next_page_token'].get('id', 0) }}"
+          properties: "{{ stream_slice.extra_fields['query_properties'] }}"
+          after: "{{ next_page_token['next_page_token']['after'] }}"
+        request_parameters:
```

Comment:
based on api docs, I don't think the `request_params` are utilized in the request since this a POST request so we can probably remove this?